README.md

```diff
@@ -1,15 +1,17 @@
+[![Build Status](https://travis-ci.org/RUGSoftEng/2017-Hestia-Client.svg?branch=development)](https://travis-ci.org/RUGSoftEng/2017-Hestia-Client)
+
 # 2017-Hestia
 A home automation system.
 
-## General Description 
+## General Description
 This repository contains the clientside code of the **Hestia** home automation system. The system consists of a server which has to be set up at home and an app.
 The server is used to manage all home automation devices. The app, which is available as an apk for the Android framework,
-connects to the server and provides an interface to control the devices. 
+connects to the server and provides an interface to control the devices.
 
-The current code represents a Minimal Viable Product (MVP), which supports the Philips Hue system. We will extend this system in the near future to add support for more plugins. 
+The current code represents a Minimal Viable Product (MVP), which supports the Philips Hue system. We will extend this system in the near future to add support for more plugins.
 
 ## Setup
-To install the app, all that is needed is the apk. The apk can be obtained by pulling the current release (0.3) from the master branch and then transferring the apk to the Android device. Alternatively, any version of the apk can be manually compiled using Android Studio. All you need to do is switch to the appropriate branch, open the project in Android Studio and compile. 
+To install the app, all that is needed is the apk. The apk can be obtained by pulling the current release (0.3) from the master branch and then transferring the apk to the Android device. Alternatively, any version of the apk can be manually compiled using Android Studio. All you need to do is switch to the appropriate branch, open the project in Android Studio and compile.
 ## Usage
```

Comment:
About documentation: I would like to have the javadocs (or a better alternative doctool if you have that) hosted on gh-pages. Ask me if you do not understand how gh-pages works, I will help you.